agent/agents/process/process_logger.go

```diff
@@ -116,6 +117,27 @@ func replacer(redactWords []string) *strings.Replacer {
 	return strings.NewReplacer(r...)
 }
 
+var matchLogLevelRegex = regexp.MustCompile(`level=(\w+)`)
+
+func matchLogLevel(line string) logrus.Level {
+	matches := matchLogLevelRegex.FindStringSubmatch(line)
+	if len(matches) < 2 {
```

Comment:
Already exists in Go documentation https://pkg.go.dev/regexp#Regexp.FindStringSubmatch

```go
func main() {
	re := regexp.MustCompile(`a(x*)b(y|z)c`)
	fmt.Printf("%q\n", re.FindStringSubmatch("-axxxbyc-"))
	fmt.Printf("%q\n", re.FindStringSubmatch("-abzc-"))
}
```
```text
["axxxbyc" "xxx" "y"]
["abzc" "" "z"]
```

Reply:
@qwest812 instead of writing comments it is better to refactor

@YaroslavPodorvanov can you please extract `len(matches) < 2` into a var and can move to a a `defaultLogLevel` constant

```
const defaultLogLevel = logrus.InfoLevel

...
noMatches := len(matches) < 2
if noMatches {
  return defaultLogLevel
}
```

Reply:
done